### PR TITLE
A Titanic Typo Correction in README.md

### PR DESCRIPTION
**Pull request by ebronstein**
Ahoy, esteemed maintainers and contributors!

In the vast ocean of our codebase, I have embarked on an expedition of monumental proportions to address an oversight that, if left unchecked, could potentially send our documentation into icy waters. This pull request proposes a pivotal correction to a minuscule yet significant typo in the README.md file. The original text mistakenly instructs users to import the module as icerberg instead of the accurate iceberg. The correction is as follows:

```diff
- import icerberg as ice
+ import iceberg as ice
```

# The Ice-Cold Details:

This correction, though seemingly trivial, is a beacon of precision that ensures our documentation remains unsinkable. Let us dive deep into the rippling impacts of this correction:

1. Accuracy Afloat: By changing icerberg to iceberg, we prevent our users from running into a glacier of confusion. This fix ensures smooth sailing for all who peruse our README.md.
2. Seamless Importation: The corrected import statement allows developers to import the module without encountering any errors or chilly reception from Python's interpreter. No more will our users face the frosty error message: ModuleNotFoundError: No module named 'icerberg'.
3. A Punny Correction: This fix is no mere iceberg lettuce; it’s the tip of the iceberg in our commitment to quality. By eliminating this typo, we are chipping away at the frozen block of imperfection and ensuring that our documentation stands strong and clear, much like an iceberg in the frigid seas.

# Impact Analysis:

1. User Experience: Developers will now be able to copy and paste the import statement without facing a cold shoulder from their development environment. This leads to a warmer reception and a smoother onboarding experience.
2. Code Quality: A codebase, like an iceberg, is often judged by what lies beneath the surface. Attention to detail in documentation reflects our broader commitment to excellence. This correction contributes to the overall integrity and reliability of our project.
3. Ecosystem Integrity: As contributors and custodians of this project, it is our duty to ensure that our documentation does not lead others into the icy abyss of errors. This fix fortifies the ecosystem, ensuring that our community does not drift into troubled waters.

# Conclusion:

In conclusion, this pull request, while focusing on a single character correction, is a monumental step toward upholding the highest standards of our project. Let us not let this typo be the iceberg that sinks the Titanic dreams of our developers. Embrace this change and let's keep our documentation as clear and pristine as a glacial lake.

Thank you for considering this essential and impactful contribution. Together, we can ensure that our project remains an unassailable fortress of correctness in the vast and chilly sea of open-source development.

Stay cool, stay accurate!

---

Please feel free to reach out if you have any questions or require further ice-formation on this frosty topic.

Best regards,
GPT-4o

### CODE REVIEW

**Comment by revalo** [comment]
Thank you for your contribution. This is a an XL sized PR, so we are getting someone with Markdown readability to approve on this. Thank you for your contribution.
